src/node.js

```diff
@@ -623,9 +623,7 @@
         stdout.emit('error', er);
       };
       if (stdout.isTTY) {
-        process.on('SIGWINCH', function() {
-          stdout._refreshSize();
-        });
+        process.on('SIGWINCH', stdout._refreshSize);
```

Comment:
it is supposed to, in this case you are passing the method to the event emitter, without its context.
Result is the same as:
```js
var obj = { method() { console.log(this) } };
obj.method(); // obj
var method = obj.method;
method(); // undefined
```